server/server_test.go

```diff
@@ -45,10 +45,12 @@ const cborFile = "../testdata/certs/cbor/ecdsap256_nosct.cbor"
 
 func setupServer(www *httptest.Server) (*server.Server, string) {
 	ac := certchaintest.MustReadAugmentedChainFile(cborFile)
+	cache := newStubCache()
 
 	certManager := certmanager.NewManager(certmanager.Config{
 		RawChainSource: &stubRawChainSource{ac.RawChain},
 		OCSPRespSource: &stubOCSPRespSource{ac.OCSPResp},
+		Cache:          cache,
```

Comment:
You can call `newStubCache` straight here.

Reply:
Done